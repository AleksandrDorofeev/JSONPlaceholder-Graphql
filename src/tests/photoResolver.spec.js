const { createTestClient } = require("apollo-server-testing");
const {
  ApolloServer,
  makeExecutableSchema,
  gql,
} = require("apollo-server-express");

const typeDefs = require("../schema");
const resolvers = require("../resolver");

const GET_ALL_PHOTOS = gql`
  query GetAllPhotos {
    photos {
      albumId
      id
      title
      url
      thumbnailUrl
    }
  }
`;

const GET_ONE_PHOTO = gql`
  query GetOnePhoto($id: ID!) {
    photo(id: $id) {
      ... on PhotoNotFound {
        error
        message
      }
      ... on Photo {
        albumId
        id
        title
        url
        thumbnailUrl
      }
    }
  }
`;

const constructTestServer = () => {
  const schema = makeExecutableSchema({
    typeDefs,
    resolvers,
    resolverValidationOptions: {
      requireResolversForResolveType: false,
    },
  });

  const server = new ApolloServer({
    schema,
  });

  return { server };
};

describe("[testing photo queries]", () => {
  it("GET_ALL_PHOTOS", async () => {
    const { server } = constructTestServer();
    const { query } = createTestClient(server);

    const res = await query({
      query: GET_ALL_PHOTOS,
    });

    expect(res).toMatchSnapshot();
  });

  it("GET_ONE_PHOTO with existing photo", async () => {
    const { server } = constructTestServer();
    const { query } = createTestClient(server);

    const res = await query({
      query: GET_ONE_PHOTO,
      variables: { id: 5 },
    });

    expect(res).toMatchSnapshot();
  });

  it("GET_ONE_PHOTO with not existing photo", async () => {
    const { server } = constructTestServer();
    const { query } = createTestClient(server);

    const res = await query({
      query: GET_ONE_PHOTO,
      variables: { id: 500 },
    });

    expect(res).toMatchSnapshot();
  });
});
