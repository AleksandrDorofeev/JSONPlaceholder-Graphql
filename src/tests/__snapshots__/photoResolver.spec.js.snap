// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[testing photo queries] GET_ALL_PHOTOS 1`] = `
Object {
  "data": Object {
    "photos": Array [
      Object {
        "albumId": "1",
        "id": "1",
        "thumbnailUrl": "https://via.placeholder.com/150/92c952",
        "title": "accusamus beatae ad facilis cum similique qui sunt",
        "url": "https://via.placeholder.com/600/92c952",
      },
      Object {
        "albumId": "1",
        "id": "2",
        "thumbnailUrl": "https://via.placeholder.com/150/771796",
        "title": "reprehenderit est deserunt velit ipsam",
        "url": "https://via.placeholder.com/600/771796",
      },
      Object {
        "albumId": "1",
        "id": "3",
        "thumbnailUrl": "https://via.placeholder.com/150/24f355",
        "title": "officia porro iure quia iusto qui ipsa ut modi",
        "url": "https://via.placeholder.com/600/24f355",
      },
      Object {
        "albumId": "1",
        "id": "4",
        "thumbnailUrl": "https://via.placeholder.com/150/d32776",
        "title": "culpa odio esse rerum omnis laboriosam voluptate repudiandae",
        "url": "https://via.placeholder.com/600/d32776",
      },
      Object {
        "albumId": "1",
        "id": "5",
        "thumbnailUrl": "https://via.placeholder.com/150/f66b97",
        "title": "natus nisi omnis corporis facere molestiae rerum in",
        "url": "https://via.placeholder.com/600/f66b97",
      },
      Object {
        "albumId": "1",
        "id": "6",
        "thumbnailUrl": "https://via.placeholder.com/150/56a8c2",
        "title": "accusamus ea aliquid et amet sequi nemo",
        "url": "https://via.placeholder.com/600/56a8c2",
      },
      Object {
        "albumId": "1",
        "id": "7",
        "thumbnailUrl": "https://via.placeholder.com/150/b0f7cc",
        "title": "officia delectus consequatur vero aut veniam explicabo molestias",
        "url": "https://via.placeholder.com/600/b0f7cc",
      },
      Object {
        "albumId": "1",
        "id": "8",
        "thumbnailUrl": "https://via.placeholder.com/150/54176f",
        "title": "aut porro officiis laborum odit ea laudantium corporis",
        "url": "https://via.placeholder.com/600/54176f",
      },
      Object {
        "albumId": "1",
        "id": "9",
        "thumbnailUrl": "https://via.placeholder.com/150/51aa97",
        "title": "qui eius qui autem sed",
        "url": "https://via.placeholder.com/600/51aa97",
      },
      Object {
        "albumId": "1",
        "id": "10",
        "thumbnailUrl": "https://via.placeholder.com/150/810b14",
        "title": "beatae et provident et ut vel",
        "url": "https://via.placeholder.com/600/810b14",
      },
      Object {
        "albumId": "1",
        "id": "11",
        "thumbnailUrl": "https://via.placeholder.com/150/1ee8a4",
        "title": "nihil at amet non hic quia qui",
        "url": "https://via.placeholder.com/600/1ee8a4",
      },
      Object {
        "albumId": "1",
        "id": "12",
        "thumbnailUrl": "https://via.placeholder.com/150/66b7d2",
        "title": "mollitia soluta ut rerum eos aliquam consequatur perspiciatis maiores",
        "url": "https://via.placeholder.com/600/66b7d2",
      },
      Object {
        "albumId": "1",
        "id": "13",
        "thumbnailUrl": "https://via.placeholder.com/150/197d29",
        "title": "repudiandae iusto deleniti rerum",
        "url": "https://via.placeholder.com/600/197d29",
      },
      Object {
        "albumId": "1",
        "id": "14",
        "thumbnailUrl": "https://via.placeholder.com/150/61a65",
        "title": "est necessitatibus architecto ut laborum",
        "url": "https://via.placeholder.com/600/61a65",
      },
      Object {
        "albumId": "1",
        "id": "15",
        "thumbnailUrl": "https://via.placeholder.com/150/f9cee5",
        "title": "harum dicta similique quis dolore earum ex qui",
        "url": "https://via.placeholder.com/600/f9cee5",
      },
      Object {
        "albumId": "1",
        "id": "16",
        "thumbnailUrl": "https://via.placeholder.com/150/fdf73e",
        "title": "iusto sunt nobis quasi veritatis quas expedita voluptatum deserunt",
        "url": "https://via.placeholder.com/600/fdf73e",
      },
      Object {
        "albumId": "1",
        "id": "17",
        "thumbnailUrl": "https://via.placeholder.com/150/9c184f",
        "title": "natus doloribus necessitatibus ipsa",
        "url": "https://via.placeholder.com/600/9c184f",
      },
      Object {
        "albumId": "1",
        "id": "18",
        "thumbnailUrl": "https://via.placeholder.com/150/1fe46f",
        "title": "laboriosam odit nam necessitatibus et illum dolores reiciendis",
        "url": "https://via.placeholder.com/600/1fe46f",
      },
      Object {
        "albumId": "1",
        "id": "19",
        "thumbnailUrl": "https://via.placeholder.com/150/56acb2",
        "title": "perferendis nesciunt eveniet et optio a",
        "url": "https://via.placeholder.com/600/56acb2",
      },
      Object {
        "albumId": "1",
        "id": "20",
        "thumbnailUrl": "https://via.placeholder.com/150/8985dc",
        "title": "assumenda voluptatem laboriosam enim consequatur veniam placeat reiciendis error",
        "url": "https://via.placeholder.com/600/8985dc",
      },
      Object {
        "albumId": "1",
        "id": "21",
        "thumbnailUrl": "https://via.placeholder.com/150/5e12c6",
        "title": "ad et natus qui",
        "url": "https://via.placeholder.com/600/5e12c6",
      },
      Object {
        "albumId": "1",
        "id": "22",
        "thumbnailUrl": "https://via.placeholder.com/150/45601a",
        "title": "et ea illo et sit voluptas animi blanditiis porro",
        "url": "https://via.placeholder.com/600/45601a",
      },
      Object {
        "albumId": "1",
        "id": "23",
        "thumbnailUrl": "https://via.placeholder.com/150/e924e6",
        "title": "harum velit vero totam",
        "url": "https://via.placeholder.com/600/e924e6",
      },
      Object {
        "albumId": "1",
        "id": "24",
        "thumbnailUrl": "https://via.placeholder.com/150/8f209a",
        "title": "beatae officiis ut aut",
        "url": "https://via.placeholder.com/600/8f209a",
      },
      Object {
        "albumId": "1",
        "id": "25",
        "thumbnailUrl": "https://via.placeholder.com/150/5e3a73",
        "title": "facere non quis fuga fugit vitae",
        "url": "https://via.placeholder.com/600/5e3a73",
      },
      Object {
        "albumId": "1",
        "id": "26",
        "thumbnailUrl": "https://via.placeholder.com/150/474645",
        "title": "asperiores nobis voluptate qui",
        "url": "https://via.placeholder.com/600/474645",
      },
      Object {
        "albumId": "1",
        "id": "27",
        "thumbnailUrl": "https://via.placeholder.com/150/c984bf",
        "title": "sit asperiores est quos quis nisi veniam error",
        "url": "https://via.placeholder.com/600/c984bf",
      },
      Object {
        "albumId": "1",
        "id": "28",
        "thumbnailUrl": "https://via.placeholder.com/150/392537",
        "title": "non neque eligendi molestiae repudiandae illum voluptatem qui aut",
        "url": "https://via.placeholder.com/600/392537",
      },
      Object {
        "albumId": "1",
        "id": "29",
        "thumbnailUrl": "https://via.placeholder.com/150/602b9e",
        "title": "aut ipsam quos ab placeat omnis",
        "url": "https://via.placeholder.com/600/602b9e",
      },
      Object {
        "albumId": "1",
        "id": "30",
        "thumbnailUrl": "https://via.placeholder.com/150/372c93",
        "title": "odio enim voluptatem quidem aut nihil illum",
        "url": "https://via.placeholder.com/600/372c93",
      },
      Object {
        "albumId": "1",
        "id": "31",
        "thumbnailUrl": "https://via.placeholder.com/150/a7c272",
        "title": "voluptate voluptates sequi",
        "url": "https://via.placeholder.com/600/a7c272",
      },
      Object {
        "albumId": "1",
        "id": "32",
        "thumbnailUrl": "https://via.placeholder.com/150/c70a4d",
        "title": "ad enim dignissimos voluptatem similique",
        "url": "https://via.placeholder.com/600/c70a4d",
      },
      Object {
        "albumId": "1",
        "id": "33",
        "thumbnailUrl": "https://via.placeholder.com/150/501fe1",
        "title": "culpa ipsam nobis qui fuga magni et mollitia",
        "url": "https://via.placeholder.com/600/501fe1",
      },
      Object {
        "albumId": "1",
        "id": "34",
        "thumbnailUrl": "https://via.placeholder.com/150/35185e",
        "title": "vitae est facere quia itaque adipisci perferendis id maiores",
        "url": "https://via.placeholder.com/600/35185e",
      },
      Object {
        "albumId": "1",
        "id": "35",
        "thumbnailUrl": "https://via.placeholder.com/150/c96cad",
        "title": "tenetur minus voluptatum et",
        "url": "https://via.placeholder.com/600/c96cad",
      },
      Object {
        "albumId": "1",
        "id": "36",
        "thumbnailUrl": "https://via.placeholder.com/150/4d564d",
        "title": "expedita rerum eaque",
        "url": "https://via.placeholder.com/600/4d564d",
      },
      Object {
        "albumId": "1",
        "id": "37",
        "thumbnailUrl": "https://via.placeholder.com/150/ea51da",
        "title": "totam voluptas iusto deserunt dolores",
        "url": "https://via.placeholder.com/600/ea51da",
      },
      Object {
        "albumId": "1",
        "id": "38",
        "thumbnailUrl": "https://via.placeholder.com/150/4f5b8d",
        "title": "natus magnam iure rerum pariatur molestias dolore nisi",
        "url": "https://via.placeholder.com/600/4f5b8d",
      },
      Object {
        "albumId": "1",
        "id": "39",
        "thumbnailUrl": "https://via.placeholder.com/150/1e71a2",
        "title": "molestiae nam ullam et rerum doloribus",
        "url": "https://via.placeholder.com/600/1e71a2",
      },
      Object {
        "albumId": "1",
        "id": "40",
        "thumbnailUrl": "https://via.placeholder.com/150/3a0b95",
        "title": "est quas voluptates dignissimos sint praesentium nisi recusandae",
        "url": "https://via.placeholder.com/600/3a0b95",
      },
      Object {
        "albumId": "1",
        "id": "41",
        "thumbnailUrl": "https://via.placeholder.com/150/659403",
        "title": "in voluptatem doloremque cum atque architecto deleniti",
        "url": "https://via.placeholder.com/600/659403",
      },
      Object {
        "albumId": "1",
        "id": "42",
        "thumbnailUrl": "https://via.placeholder.com/150/ca50ac",
        "title": "voluptatibus a autem molestias voluptas architecto culpa",
        "url": "https://via.placeholder.com/600/ca50ac",
      },
      Object {
        "albumId": "1",
        "id": "43",
        "thumbnailUrl": "https://via.placeholder.com/150/6ad437",
        "title": "eius hic autem ad beatae voluptas",
        "url": "https://via.placeholder.com/600/6ad437",
      },
      Object {
        "albumId": "1",
        "id": "44",
        "thumbnailUrl": "https://via.placeholder.com/150/29fe9f",
        "title": "neque eum provident et inventore sed ipsam dignissimos quo",
        "url": "https://via.placeholder.com/600/29fe9f",
      },
      Object {
        "albumId": "1",
        "id": "45",
        "thumbnailUrl": "https://via.placeholder.com/150/c4084a",
        "title": "praesentium fugit quis aut voluptatum commodi dolore corrupti",
        "url": "https://via.placeholder.com/600/c4084a",
      },
      Object {
        "albumId": "1",
        "id": "46",
        "thumbnailUrl": "https://via.placeholder.com/150/e9b68",
        "title": "quidem maiores in quia fugit dolore explicabo occaecati",
        "url": "https://via.placeholder.com/600/e9b68",
      },
      Object {
        "albumId": "1",
        "id": "47",
        "thumbnailUrl": "https://via.placeholder.com/150/b4412f",
        "title": "et soluta est",
        "url": "https://via.placeholder.com/600/b4412f",
      },
      Object {
        "albumId": "1",
        "id": "48",
        "thumbnailUrl": "https://via.placeholder.com/150/68e0a8",
        "title": "ut esse id",
        "url": "https://via.placeholder.com/600/68e0a8",
      },
      Object {
        "albumId": "1",
        "id": "49",
        "thumbnailUrl": "https://via.placeholder.com/150/2cd88b",
        "title": "quasi quae est modi quis quam in impedit",
        "url": "https://via.placeholder.com/600/2cd88b",
      },
      Object {
        "albumId": "1",
        "id": "50",
        "thumbnailUrl": "https://via.placeholder.com/150/9e59da",
        "title": "et inventore quae ut tempore eius voluptatum",
        "url": "https://via.placeholder.com/600/9e59da",
      },
      Object {
        "albumId": "2",
        "id": "51",
        "thumbnailUrl": "https://via.placeholder.com/150/8e973b",
        "title": "non sunt voluptatem placeat consequuntur rem incidunt",
        "url": "https://via.placeholder.com/600/8e973b",
      },
      Object {
        "albumId": "2",
        "id": "52",
        "thumbnailUrl": "https://via.placeholder.com/150/121fa4",
        "title": "eveniet pariatur quia nobis reiciendis laboriosam ea",
        "url": "https://via.placeholder.com/600/121fa4",
      },
      Object {
        "albumId": "2",
        "id": "53",
        "thumbnailUrl": "https://via.placeholder.com/150/6efc5f",
        "title": "soluta et harum aliquid officiis ab omnis consequatur",
        "url": "https://via.placeholder.com/600/6efc5f",
      },
      Object {
        "albumId": "2",
        "id": "54",
        "thumbnailUrl": "https://via.placeholder.com/150/aa8f2e",
        "title": "ut ex quibusdam dolore mollitia",
        "url": "https://via.placeholder.com/600/aa8f2e",
      },
      Object {
        "albumId": "2",
        "id": "55",
        "thumbnailUrl": "https://via.placeholder.com/150/5e04a4",
        "title": "voluptatem consequatur totam qui aut iure est vel",
        "url": "https://via.placeholder.com/600/5e04a4",
      },
      Object {
        "albumId": "2",
        "id": "56",
        "thumbnailUrl": "https://via.placeholder.com/150/f9f067",
        "title": "vel voluptatem esse consequuntur est officia quo aut quisquam",
        "url": "https://via.placeholder.com/600/f9f067",
      },
      Object {
        "albumId": "2",
        "id": "57",
        "thumbnailUrl": "https://via.placeholder.com/150/95acce",
        "title": "vero est optio expedita quis ut molestiae",
        "url": "https://via.placeholder.com/600/95acce",
      },
      Object {
        "albumId": "2",
        "id": "58",
        "thumbnailUrl": "https://via.placeholder.com/150/cde4c1",
        "title": "rem pariatur facere eaque",
        "url": "https://via.placeholder.com/600/cde4c1",
      },
      Object {
        "albumId": "2",
        "id": "59",
        "thumbnailUrl": "https://via.placeholder.com/150/a46a91",
        "title": "modi totam dolor eaque et ipsum est cupiditate",
        "url": "https://via.placeholder.com/600/a46a91",
      },
      Object {
        "albumId": "2",
        "id": "60",
        "thumbnailUrl": "https://via.placeholder.com/150/323599",
        "title": "ea enim temporibus asperiores placeat consectetur commodi ullam",
        "url": "https://via.placeholder.com/600/323599",
      },
      Object {
        "albumId": "2",
        "id": "61",
        "thumbnailUrl": "https://via.placeholder.com/150/e403d1",
        "title": "quia minus sed eveniet accusantium incidunt beatae odio",
        "url": "https://via.placeholder.com/600/e403d1",
      },
      Object {
        "albumId": "2",
        "id": "62",
        "thumbnailUrl": "https://via.placeholder.com/150/65ad4f",
        "title": "dolorem cumque quo nihil inventore enim",
        "url": "https://via.placeholder.com/600/65ad4f",
      },
      Object {
        "albumId": "2",
        "id": "63",
        "thumbnailUrl": "https://via.placeholder.com/150/4e557c",
        "title": "facere animi autem quod dolor",
        "url": "https://via.placeholder.com/600/4e557c",
      },
      Object {
        "albumId": "2",
        "id": "64",
        "thumbnailUrl": "https://via.placeholder.com/150/cd5a92",
        "title": "doloremque culpa quia",
        "url": "https://via.placeholder.com/600/cd5a92",
      },
      Object {
        "albumId": "2",
        "id": "65",
        "thumbnailUrl": "https://via.placeholder.com/150/149540",
        "title": "sed voluptatum enim eaque cumque qui sunt",
        "url": "https://via.placeholder.com/600/149540",
      },
      Object {
        "albumId": "2",
        "id": "66",
        "thumbnailUrl": "https://via.placeholder.com/150/ee0a7e",
        "title": "provident rerum voluptatem illo asperiores qui maiores",
        "url": "https://via.placeholder.com/600/ee0a7e",
      },
      Object {
        "albumId": "2",
        "id": "67",
        "thumbnailUrl": "https://via.placeholder.com/150/1279e9",
        "title": "veritatis labore ipsum unde aut quam dolores",
        "url": "https://via.placeholder.com/600/1279e9",
      },
      Object {
        "albumId": "2",
        "id": "68",
        "thumbnailUrl": "https://via.placeholder.com/150/e9603b",
        "title": "architecto aut quod qui ullam vitae expedita delectus",
        "url": "https://via.placeholder.com/600/e9603b",
      },
      Object {
        "albumId": "2",
        "id": "69",
        "thumbnailUrl": "https://via.placeholder.com/150/46e3b1",
        "title": "et autem dolores aut porro est qui",
        "url": "https://via.placeholder.com/600/46e3b1",
      },
      Object {
        "albumId": "2",
        "id": "70",
        "thumbnailUrl": "https://via.placeholder.com/150/7375af",
        "title": "quam quos dolor eum ea in",
        "url": "https://via.placeholder.com/600/7375af",
      },
      Object {
        "albumId": "2",
        "id": "71",
        "thumbnailUrl": "https://via.placeholder.com/150/363789",
        "title": "illo qui vel laboriosam vel fugit deserunt",
        "url": "https://via.placeholder.com/600/363789",
      },
      Object {
        "albumId": "2",
        "id": "72",
        "thumbnailUrl": "https://via.placeholder.com/150/45935c",
        "title": "iusto sint enim nesciunt facilis exercitationem",
        "url": "https://via.placeholder.com/600/45935c",
      },
      Object {
        "albumId": "2",
        "id": "73",
        "thumbnailUrl": "https://via.placeholder.com/150/1224bd",
        "title": "rerum exercitationem libero dolor",
        "url": "https://via.placeholder.com/600/1224bd",
      },
      Object {
        "albumId": "2",
        "id": "74",
        "thumbnailUrl": "https://via.placeholder.com/150/65ac19",
        "title": "eligendi quas consequatur aut consequuntur",
        "url": "https://via.placeholder.com/600/65ac19",
      },
      Object {
        "albumId": "2",
        "id": "75",
        "thumbnailUrl": "https://via.placeholder.com/150/a9ef52",
        "title": "aut magni quibusdam cupiditate ea",
        "url": "https://via.placeholder.com/600/a9ef52",
      },
      Object {
        "albumId": "2",
        "id": "76",
        "thumbnailUrl": "https://via.placeholder.com/150/7644fe",
        "title": "magni nulla et dolores",
        "url": "https://via.placeholder.com/600/7644fe",
      },
      Object {
        "albumId": "2",
        "id": "77",
        "thumbnailUrl": "https://via.placeholder.com/150/36d137",
        "title": "ipsum consequatur vel omnis mollitia repellat dolores quasi",
        "url": "https://via.placeholder.com/600/36d137",
      },
      Object {
        "albumId": "2",
        "id": "78",
        "thumbnailUrl": "https://via.placeholder.com/150/637984",
        "title": "aperiam aut est amet tenetur et dolorem",
        "url": "https://via.placeholder.com/600/637984",
      },
      Object {
        "albumId": "2",
        "id": "79",
        "thumbnailUrl": "https://via.placeholder.com/150/c611a9",
        "title": "est vel et laboriosam quo aspernatur distinctio molestiae",
        "url": "https://via.placeholder.com/600/c611a9",
      },
      Object {
        "albumId": "2",
        "id": "80",
        "thumbnailUrl": "https://via.placeholder.com/150/a0c998",
        "title": "et corrupti nihil cumque",
        "url": "https://via.placeholder.com/600/a0c998",
      },
      Object {
        "albumId": "2",
        "id": "81",
        "thumbnailUrl": "https://via.placeholder.com/150/31a74c",
        "title": "error magni fugiat dolorem impedit molestiae illo ullam debitis",
        "url": "https://via.placeholder.com/600/31a74c",
      },
      Object {
        "albumId": "2",
        "id": "82",
        "thumbnailUrl": "https://via.placeholder.com/150/88b703",
        "title": "voluptate voluptas molestias vitae illo iusto",
        "url": "https://via.placeholder.com/600/88b703",
      },
      Object {
        "albumId": "2",
        "id": "83",
        "thumbnailUrl": "https://via.placeholder.com/150/a19891",
        "title": "quia quasi enim voluptatem repellat sit sint",
        "url": "https://via.placeholder.com/600/a19891",
      },
      Object {
        "albumId": "2",
        "id": "84",
        "thumbnailUrl": "https://via.placeholder.com/150/b5205d",
        "title": "aliquam dolorem ut modi ratione et assumenda impedit",
        "url": "https://via.placeholder.com/600/b5205d",
      },
      Object {
        "albumId": "2",
        "id": "85",
        "thumbnailUrl": "https://via.placeholder.com/150/eb7e7f",
        "title": "ullam delectus architecto sint error",
        "url": "https://via.placeholder.com/600/eb7e7f",
      },
      Object {
        "albumId": "2",
        "id": "86",
        "thumbnailUrl": "https://via.placeholder.com/150/fd5751",
        "title": "qui vel ut odio consequuntur",
        "url": "https://via.placeholder.com/600/fd5751",
      },
      Object {
        "albumId": "2",
        "id": "87",
        "thumbnailUrl": "https://via.placeholder.com/150/224566",
        "title": "eos nihil sunt accusantium omnis",
        "url": "https://via.placeholder.com/600/224566",
      },
      Object {
        "albumId": "2",
        "id": "88",
        "thumbnailUrl": "https://via.placeholder.com/150/75334a",
        "title": "inventore veritatis magnam enim quasi",
        "url": "https://via.placeholder.com/600/75334a",
      },
      Object {
        "albumId": "2",
        "id": "89",
        "thumbnailUrl": "https://via.placeholder.com/150/21d35",
        "title": "id at cum incidunt nulla dolor vero tenetur",
        "url": "https://via.placeholder.com/600/21d35",
      },
      Object {
        "albumId": "2",
        "id": "90",
        "thumbnailUrl": "https://via.placeholder.com/150/bfe0dc",
        "title": "et quae eligendi vitae maxime in",
        "url": "https://via.placeholder.com/600/bfe0dc",
      },
      Object {
        "albumId": "2",
        "id": "91",
        "thumbnailUrl": "https://via.placeholder.com/150/40591",
        "title": "sunt quo laborum commodi porro consequatur nam delectus et",
        "url": "https://via.placeholder.com/600/40591",
      },
      Object {
        "albumId": "2",
        "id": "92",
        "thumbnailUrl": "https://via.placeholder.com/150/de79c7",
        "title": "quod non quae",
        "url": "https://via.placeholder.com/600/de79c7",
      },
      Object {
        "albumId": "2",
        "id": "93",
        "thumbnailUrl": "https://via.placeholder.com/150/2edde0",
        "title": "molestias et aliquam natus repellendus accusamus dolore",
        "url": "https://via.placeholder.com/600/2edde0",
      },
      Object {
        "albumId": "2",
        "id": "94",
        "thumbnailUrl": "https://via.placeholder.com/150/cc12f5",
        "title": "et quisquam aspernatur",
        "url": "https://via.placeholder.com/600/cc12f5",
      },
      Object {
        "albumId": "2",
        "id": "95",
        "thumbnailUrl": "https://via.placeholder.com/150/9cda61",
        "title": "magni odio non",
        "url": "https://via.placeholder.com/600/9cda61",
      },
      Object {
        "albumId": "2",
        "id": "96",
        "thumbnailUrl": "https://via.placeholder.com/150/1fb08b",
        "title": "dolore esse a in eos sed",
        "url": "https://via.placeholder.com/600/1fb08b",
      },
      Object {
        "albumId": "2",
        "id": "97",
        "thumbnailUrl": "https://via.placeholder.com/150/e2223e",
        "title": "labore magnam officiis nemo et",
        "url": "https://via.placeholder.com/600/e2223e",
      },
      Object {
        "albumId": "2",
        "id": "98",
        "thumbnailUrl": "https://via.placeholder.com/150/a77d08",
        "title": "sed commodi libero id nesciunt modi vitae",
        "url": "https://via.placeholder.com/600/a77d08",
      },
      Object {
        "albumId": "2",
        "id": "99",
        "thumbnailUrl": "https://via.placeholder.com/150/b04f2e",
        "title": "magnam dolor sed enim vel optio consequuntur",
        "url": "https://via.placeholder.com/600/b04f2e",
      },
      Object {
        "albumId": "2",
        "id": "100",
        "thumbnailUrl": "https://via.placeholder.com/150/14ba42",
        "title": "et qui rerum",
        "url": "https://via.placeholder.com/600/14ba42",
      },
    ],
  },
  "errors": undefined,
  "extensions": undefined,
  "http": Object {
    "headers": Headers {
      Symbol(map): Object {},
    },
  },
}
`;

exports[`[testing photo queries] GET_ONE_PHOTO with existing photo 1`] = `
Object {
  "data": Object {
    "photo": Object {
      "albumId": "1",
      "id": "5",
      "thumbnailUrl": "https://via.placeholder.com/150/f66b97",
      "title": "natus nisi omnis corporis facere molestiae rerum in",
      "url": "https://via.placeholder.com/600/f66b97",
    },
  },
  "errors": undefined,
  "extensions": undefined,
  "http": Object {
    "headers": Headers {
      Symbol(map): Object {},
    },
  },
}
`;

exports[`[testing photo queries] GET_ONE_PHOTO with not existing photo 1`] = `
Object {
  "data": Object {
    "photo": Object {
      "error": "Error",
      "message": "Photo with the id 500 not exists",
    },
  },
  "errors": undefined,
  "extensions": undefined,
  "http": Object {
    "headers": Headers {
      Symbol(map): Object {},
    },
  },
}
`;
